doc/source/examples/cd_clf_adult.ipynb

```diff
@@ -0,0 +1,434 @@
+{
```

Comment:
<p>Can we add a paragraph describing what this notebook is showcasing (i.e. classifier based drift detection using sklearn model and additional calibration). Also worth briefly mentioning how the drifted datasets are constructed later on.</p><p> </p><p>Also add a sentence about <code>alibi</code> needed for the <code>adult</code> dataset.</p><p> </p><p>Finally, this notebook requires a level 1 #Heading and all other headings should be level 2 ## or lower, otherwise the rendering on the docs is off: https://seldon--414.org.readthedocs.build/projects/alibi-detect/en/414/examples/cd_clf_adult.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>